modules/preview-react/loading-sparkles/stories/examples/Basic.tsx

```diff
@@ -1,6 +1,70 @@
 import React from 'react';
+import {SecondaryButton} from '@workday/canvas-kit-react/button';
+import {Text} from '@workday/canvas-kit-react/text';
 import {LoadingSparkles} from '@workday/canvas-kit-preview-react/loading-sparkles';
+import {AccessibleHide, AriaLiveRegion} from '@workday/canvas-kit-react/common';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const containerStyles = createStyles({
+  minHeight: '3.5rem',
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '0.5rem',
+});
 
 export const Basic = () => {
-  return <LoadingSparkles aria-live="polite" aria-label="Generating content" />;
+  const [isLoading, setLoading] = React.useState<'idle' | 'loading' | 'success'>('idle');
```

Comment:
does `isLoading` mean  in name to be boolean value? I would suggest `status` instead.

Reply:
Yeah, I started with a boolean. I'll update it. Good catch.